`npm install` fails

`npm install` fails.

## Environment
node: v12.9.0
npm: 6.11.1

## Log
```
> elm@0.19.0 install /Users/ikngtty/Projects/github.com/realglobe-Inc/hrafnar/node_modules/elm-webpack-loader/node_modules/elm
> binwrap-install

fs.js:27
const { Math, Object } = primordials;
                         ^

ReferenceError: primordials is not defined
    at fs.js:27:26
    at req_ (/Users/ikngtty/Projects/github.com/realglobe-Inc/hrafnar/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/Users/ikngtty/Projects/github.com/realglobe-Inc/hrafnar/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/Users/ikngtty/Projects/github.com/realglobe-Inc/hrafnar/node_modules/unzip/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
npm WARN hrafnar@1.0.0 No description
npm WARN hrafnar@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! elm@0.19.0 install: `binwrap-install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the elm@0.19.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ikngtty/.npm/_logs/2019-08-21T09_01_32_517Z-debug.log
```

## Information
* The same problem and its cause: <https://github.com/nodejs/node/issues/27422>
* The same problem and solution in Gulp: <https://www.solutionfactory.in/posts/ReferenceError-primordials-is-not-defined-in-node-Solution>
    * This page says that this bug occurs at Node.js version 12 only.

## Solution
* Downgrade my Node.js to version 11.
* Upgrade dependencies.

I'll send PR to upgrade dependencies.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
   "devDependencies": {
     "css-loader": "^2.1.1",
     "elm": "^0.19.0-no-deps",
-    "elm-webpack-loader": "^5.0.0",
+    "elm-webpack-loader": "^6.0.0",
     "html-webpack-plugin": "^3.2.0",
     "node-sass": "^4.12.0",
     "sass-loader": "^7.1.0",

